metadata_extract/text.py

```diff
@@ -17,29 +18,79 @@ def append_to_context(self, extra_context: str) -> None:
         self.context = (self.context or '') + extra_context
 
 
-files = InitFiles()
-LABELS = files.get_labels()
-STOPWORDS = files.get_stopwords()
+def labels() -> dict[str, str]:
+    return ResourceLoader.get_labels()
+
+
+def stopwords() -> list[str]:
```

Comment:
This method, as well as `info_page_keywords` and `doc_type_mapping` can just be removed. Use the static ResourceLoader method directly where needed.
`labels` is useful as a syntactic sugar for the regular expressions, but could be made private (`__labels`).